src/routes/accomodationRoutes.js

```diff
@@ -27,11 +36,17 @@ accomodationRouter.get(
   Accomodation.getById
 );
 
+accomodationRouter.get(
+  '/v1/accommodations/:accomodation_id/destination-stats',
+  isLoggedIn,
```

Comment:
You can name this route '/v1/accommodations/:accomodation_id/destinationStats' to have similar naming for routes 
You should also put validation of parameter